projects/plugins/jetpack/_inc/client/my-plan/my-plan-body.jsx

```diff
@@ -340,7 +340,9 @@ class MyPlanBody extends React.Component {
 								this.props.isPluginActive( 'akismet/akismet.php' ) ? (
 									<Button
 										onClick={ this.handleButtonClickForTracking( 'view_spam_stats' ) }
-										href={ this.props.siteAdminUrl + 'admin.php?page=akismet-key-config' }
+										href={
+											this.props.siteAdminUrl + 'admin.php?page=akismet-key-config&view=stats'
```

Comment:
Just a nitpick, can we use [Template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here?

Reply:
Thanks for the review Yashwin :-D yeah I can update that

Reply:
@yashwin I have updated the PR to use template literal